File: bidding-engine/src/main/java/com/omnibus/bidding/domain/RecommendationResponse.java
package com.omnibus.bidding.domain;

import com.omnibus.bidding.rules.CandidateBid;
import java.util.List;

public record RecommendationResponse(
    String system,
    String evaluatedSeat,
    String northHand,
    String southHand,
    String auction,
    String recommendedBid,
    String explanation,
    List<CandidateBid> candidates
) {
}

File: services/bidding-engine/src/main/kotlin/com/omnibus/bidding/domain/RecommendationResponse.kt
package com.omnibus.bidding.domain

import com.omnibus.bidding.rules.CandidateBid

data class RecommendationResponse(
    val system: String,
    val evaluatedSeat: String,
    val northHand: String,
    val southHand: String,
    val auction: String,
    val recommendedBid: String,
    val explanation: String,
    val candidates: List<CandidateBid>,
)

